packages/remix/src/client/performance.tsx

```diff
@@ -121,34 +121,6 @@ function startNavigationSpan(matches: RouteMatch<string>[]): void {
   }
 }
 
-/**
- * @deprecated Use `browserTracingIntegration` instead.
- *
- * Creates a react-router v6 instrumention for Remix applications.
- *
- * This implementation is slightly different (and simpler) from the react-router instrumentation
- * as in Remix, `useMatches` hook is available where in react-router-v6 it's not yet.
- */
-export function remixRouterInstrumentation(useEffect: UseEffect, useLocation: UseLocation, useMatches: UseMatches) {
-  return (
-    customStartTransaction: (context: TransactionContext) => Transaction | undefined,
-    startTransactionOnPageLoad = true,
-    startTransactionOnLocationChange = true,
-  ): void => {
-    setGlobals({
```

Comment:
I think we can also remove the `setGlobals` method and everything related to it then?

Reply:
`setGlobals` is still used by `browserTracingIntegration`

Reply:
ahh right, sorry.